packages/aws-cdk/test/api/hotswap/lambda-functions-hotswap-deployments.test.ts

```diff
@@ -539,3 +551,56 @@ test('does not call the updateLambdaCode() API when a resource with type that is
   expect(deployStackResult).toBeUndefined();
   expect(mockUpdateLambdaCode).not.toHaveBeenCalled();
 });
+
+test('calls getFunction() after function code is updated', async () => {
```

Comment:
Maybe let's add here "with delay 1", like we had "with delay 5" in the Docker image test?